packages/reactivated/src/generator/configuration.mts

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env node
+
+import fs from "fs";
+import path from "path";
+
+import * as esbuild from "esbuild";
+
+const CONF_INPUT_FILE = "./client/reactivated.config.tsx";
```

Comment:
Should this be `.conf` instead? 

```suggestion
const CONF_INPUT_FILE = "./client/reactivated.conf.tsx";
```

Ref: https://github.com/silviogutierrez/reactivated/pull/231#issuecomment-1897775866

Reply:
I debated this, but looked around at other JS ecosystem tools:

<img width="155" alt="image" src="https://github.com/silviogutierrez/reactivated/assets/92824/03275930-6481-4748-a07b-e44f33f4d991">

Both tailwind and capacitor use `config`. Do you have other examples?

Reply:
I don't care either way—just noted it since it differed from what you said in that comment I linked to.